.yamllint

```diff
@@ -16,13 +16,8 @@ rules:
   truthy:
     allowed-values: ["yes", "no", "true", "false", "True", "False"]
     level: error
+  # Ansible line length is 160.
```

Comment:
Why is ling length set to 160?

Reply:
IIRC this is the default for ansible-lint.

Reply:
If not set, yamllint will consider a line with more than 80 characters an error. In older versions of ansible-lint, IIRC, it warned about lines longer that 160 characters (it seems that with the most recent ones, this is not as issue anymore).

That's why I used 160 as the maximum line length. Seemed to me a good compromise in readability and ease to write playbooks. With this value, almost only certificate data need to be formatted to fit in the maximum size.